### PR TITLE
Support for Amazon Linux and Kibana 5

### PR DESCRIPTION
**Pull request by dvinograd**
Tested on Amazon Linux 2016.09 and ES/Kibana 5.1.1.

### CODE REVIEW

**Comment by drenalin23** [comment]
Thanks for submitting - this looks good.  I'll wait a little bit to see if other maintainers have any comments.  Note that a bunch of the travis tests failed, due to upstream breakage in the ark cookbook it looks like.  I am about to submit an update to fix the tests for CentOS7 as well.

**Comment by drenalin23** [comment]
@dvinograd Can you rebase or merge in the last commit with Travis.yml update and .kitchen.docker.yml fix?  That will then run all the tests again and we should be able to see if there any issues with the current test suite.

**Comment by dvinograd** [comment]
@drenalin23 done!

File: recipes/kibana4.rb [hunk 1]
```diff
@@ -52,3 +52,12 @@
   )
   notifies :restart, 'service[kibana]'
 end
+
+# Install plugins
+node['kibana']['plugins'].each do |plugin|
+  execute "install #{plugin[:name]}" do
+    command "/usr/local/kibana-#{node['kibana']['kibana4_version']}/bin/kibana plugin --install #{plugin[:url]}"
+    not_if { ::Dir.exists?("/usr/local/kibana-#{node['kibana']['kibana4_version']}/installedPlugins/#{plugin[:name]}") }
```
Comment (drenalin23) [inline recipes/kibana4.rb]: per the rubocop tests - can you update this to Dir.exist?

File: recipes/kibana5.rb [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    logging_option: node['kibana']['logging_option']
+  )
+  notifies :restart, 'service[kibana]'
+end
+
+# Install plugins
+node['kibana']['plugins'].each do |plugin|
+  execute "install #{plugin[:name]}" do
+    command "/usr/local/kibana-#{node['kibana']['kibana5_version']}/bin/kibana-plugin install #{plugin[:url]}"
+    not_if { ::Dir.exists?("/usr/local/kibana-#{node['kibana']['kibana5_version']}/plugins/#{plugin[:name]}") }
```
Comment (drenalin23) [inline recipes/kibana5.rb]: per the rubocop tests - can you update this to Dir.exist?

**Comment by drenalin23** [comment]
@dvinograd The build tests are failing too - I think because of an issue with chefdk, but haven't had time to look at it.  LMK if you have any ideas as to why those are failing - I will run a couple of local kitchen tests to see if they are passing locally.

**Comment by drenalin23** [comment]
@dvinograd the error message points at https://discourse.chef.io/t/end-of-life-announcement-for-chef-software-inc-s-bintray-account/9807
So need to figure out what components we have that are out of date...

**Comment by sbaxter** [comment]
+1

**Comment by ih16** [comment]
Hi, 

When are you planning to merge this? 

Cheers

File: recipes/kibana4.rb (hunk 1 above)
Comment (dvinograd) [inline recipes/kibana4.rb]: sorry for not following up on this.. i didn't get any notifications from Github. i'll push some updates.

Comment (dvinograd) [inline recipes/kibana4.rb]: is that the only thing that's breaking the tests?

**Comment by InformatiQ** [comment]
closing as it was integrated in #107

**Pull request closed without merging**